voodoo_sms.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |gem|
   gem.name        = 'voodoo_sms'
-  gem.version     = '0.0.2'
+  gem.version     = '0.0.3'
```

Comment:
I think this should be a major release instead of a patch as we are changing the functionality of our API